extensions/amp-inline-gallery/1.0/inline-gallery.type.js

```diff
@@ -40,3 +40,12 @@ InlineGalleryDef.Props;
  * }}
  */
 InlineGalleryDef.PaginationProps;
+
+/**
+ * @typedef {{
+ *   aspectRatio: number,
```

Comment:
Maybe we should standardize on CSS aspect-ratio format? E.g. `w/h`.

Reply:
That's what this represents, would it be helpful to annotate this in comments somewhere? Do we do this in any other type files?

Reply:
Let's document it here.

Reply:
Done